tests/integration/account/test_account.py

```diff
@@ -244,7 +244,7 @@ def get_user_id():
                 "--delimiter",
                 ",",
                 "--format",
-                "id",
+                "username",
```

Comment:
This test passed at some point because a `--format` string with an invalid field will give all of the results with username being the first. This doesn't seem to be the case anymore so I adjusted it accordingly 👍